backend/src/Squidex/Program.cs

```diff
@@ -56,6 +56,10 @@ public static void Main(string[] args)
 
                         serverOptions.ListenAnyIP(5000);
                     }
+                    else
+                    {
+                        serverOptions.ListenAnyIP(10080);
```

Comment:
Likely this will want to be expanded upon, just including in the initial changes to demonstrate how the application needs to run on a non-privileged port (1-1024) as this requires root access.

Reply:
This would be a breaking change, but you can control it with environment variables: https://andrewlock.net/5-ways-to-set-the-urls-for-an-aspnetcore-app/

Reply:
Yes, I will update this soon, I just wanted to get something up for you to see so we can discuss.

Reply:
Updated to apply from configuration.